server/container_create_linux.go

```diff
@@ -800,6 +800,11 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		return nil, err
 	}
 
+	// the existence of this file indicates that this is a container environment.
```

Comment:
this function creates a file `/run/.containerenv` on the host, but doesn't mount it into the container. You'll want to do something similar to:
https://github.com/cri-o/cri-o/blob/main/server/container_create_linux.go#L596

Reply:
@haircommander that makes sense now. I made a few changes and added a few tests. PTAL